modules/virt-creating-vm-container-disk-cli.adoc

```diff
@@ -0,0 +1,81 @@
+// Module included in the following assemblies:
+//
+// * virt/creating_vms_advanced/creating_vms_cli/virt-creating-vms-from-container-disks.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="virt-creating-vm-import-cli_{context}"]
+= Creating a VM from a container disk by using the command line
+
+You can create a virtual machine (VM) from a container disk by using the command line.
+
+.Prerequisites
+
+* You must have access credentials for the container registry that contains the container disk.
+* You must have the `virtctl` command line tool installed.
+
+.Procedure
+
+. Create a `VirtualMachine` manifest for your VM and save it as a YAML file. For example, to create a minimal {op-system-base-full} VM from a container disk, run the following command:
++
+[source,terminal]
+----
+$ virtctl create vm --name vm-rhel-9 --instancetype u1.small --preference rhel.9 --volume-containerdisk src:registry.redhat.io/rhel9/rhel-guest-image:9.5
+----
+
+. Review the `VirtualMachine` manifest for your VM:
++
+[%collapsible]
```

Comment:
This added a collapsible section, but the other modules don't have this. Should it be consistent one way or the other across them?

Reply:
That's a very good point, the collapsible attribute is inherited from the module I split. but considering the updated manifest is fairly straightforward and fits on one screen, I don't think it is needed anywhere.